### DISCUSSION TITLE
Add users

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14457] [General]
i would like a user to add other users and doesnt see any other part of admin is that possible ?
can part of admin be also given seperate access like,, permission, system setting, automation, standard module etc,,,

**Comment by rskrzypczak** [comment]
In fact, yes. Although it's not completely finished.
Check this: https://gitdeveloper.yetiforce.com/index.php?parent=Settings&module=AdminAccess&view=Index
and set the user as the super user (not admin).
Currently you will not be able to create users, only see them.

**Reply by rioshaz** [reply]
cool.. so when do u think it can be completed?
